lib/src/client_options.dart

```diff
@@ -187,6 +194,8 @@ class RestClientOptions extends ClientOptions {
     this.entitlementConfig = _defaultCacheConfig,
     this.skuConfig = _defaultCacheConfig,
     this.subscriptionConfig = _defaultCacheConfig,
+    this.globalSoundboardCacheConfig = _smallCacheConfig,
+    this.soundboardCacheConfig = _defaultCacheConfig,
```

Comment:
These should be flipped (global => default, guild => small).

Reply:
I don't agree, or at least, put the two configurations at small.
Global sounds don't seem to change frequently, we can suppose they'll never change.
And as of now, there're less than 20 global sounds.

Reply:
The guild one should definitely be small; we don't want many guilds with many stickers to bog down the bot.

The global one can currently be small, but the number of global sounds might grow. As with all caches, we don't care about a few thousand extra entities cached globally - we care about a couple hundred extra cached for every guild the client is in (which, even with just 100 guilds, is 20 000 extra entities).